src/ui/components/shared/UserSettingsModal/PreferencesSettings.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from "react";
+import { connect, ConnectedProps } from "react-redux";
+
+import { updateEnableRepaint } from "protocol/enable-repaint";
+import { handleIntercomLogout } from "ui/utils/intercom";
+import useAuth0 from "ui/utils/useAuth0";
+import hooks from "ui/hooks";
+import * as actions from "ui/actions/app";
+import * as selectors from "ui/reducers/app";
+import { UIState } from "ui/state";
+import { SettingsTabTitle } from "ui/state/app";
+import { ApiKey, UserSettings } from "ui/types";
+
+import APIKeys from "../APIKeys";
+import SettingsModal from "../SettingsModal";
+import { Settings } from "../SettingsModal/types";
+import { SettingsBodyHeader } from "../SettingsModal/SettingsBody";
+
+import { getFeatureFlag } from "ui/utils/launchdarkly";
+import { AvatarImage } from "ui/components/Avatar";
+import { prefs } from "ui/utils/prefs";
+import Checkbox from "../Forms/Checkbox";
+import { disableEmailPreference, EmailPreference, enableEmailPreference } from "ui/hooks/users";
+import Radio from "../Forms/Radio";
+
+const EMAIL_NOTIFICATIONS = {
+  [EmailPreference.COLLABORATOR_REQUESTS]: "When somebody invites you to collaborate on a replay",
+  [EmailPreference.REPLAY_COMMENTS]: "When somebody comments on your replay",
+  [EmailPreference.NEW_TEAM_INVITE]: "When you're invited to a new team",
+};
+
+function Notification({
+  content,
+  preference,
+  emailPreferences,
+}: {
+  content: string;
+  preference: EmailPreference;
+  emailPreferences: EmailPreference[];
+}) {
+  const updateUserEmailPreferences = hooks.useUpdateUserEmailPreferences();
+
+  // The preferences list is a blacklist of preferences that are disabled.
+  const checked = !emailPreferences.includes(preference);
```

Comment:
This double-negative logic is very confusing, `if it doesn't exist in the deny-list then allow it`. If I were somewhere else in the code and I saw 
```
const preferences: EmailPreference[] {
  marketing: true,
  collaboratorRequest: true
}
```

I would 100% assume that that user had opted in to those emails, but currently that would actually mean that those are the only options this user has opted *out* of? Either we should call those `EmailOptOut` or `BlockedEmail`, or we should keep the name `EmailPreference` and make it such that `true === subscribed`. This should also make it more clear how the checkboxes line up, so `true` equates to  `checked`.